sdk/servicebus/service-bus/test/batchReceiver.spec.ts

```diff
@@ -880,20 +880,6 @@ describe("Batch Receiver - Multiple Receiver Operations", function(): void {
     await firstBatchPromise;
   }
 
-  it("Partitioned Queue: Throws error when ReceiveBatch is called while the previous call is not done", async function(): Promise<
-    void
-  > {
-    await beforeEachTest(TestClientType.PartitionedQueue, TestClientType.PartitionedQueue);
-    await testParallelReceiveCalls();
-  });
-
-  it("Partitioned Subscription: Throws error when ReceiveBatch is called while the previous call is not done", async function(): Promise<
-    void
-  > {
-    await beforeEachTest(TestClientType.PartitionedTopic, TestClientType.PartitionedSubscription);
-    await testParallelReceiveCalls();
-  });
-
```

Comment:
Here, what is being tested is the behavior of the receiver on the client side. Therefore, it is enough to test a session and a non-session variation. We do need both queues & subscriptions represented. This applies to a few other places in this PR, please take a look

Reply:
Does this apply to all batch and streaming receiver tests? (Because they are all testing receiver behavior?)

On a side note, because of differences in how the AMQP protocol is implemented across languages, it feels like we should generally retain the test coverage for across entities as well.
If concern is about time taken on CI, we could annotate selective tests to run more frequently perhaps. Just concerned that we are undoing test coverage that seems otherwise useful/recommended by service team too.

Reply:
The description doesn't seem to call out on tests pertaining to non-AMQP involved / pure client side sender behavior.
I.e., there don't seem to be any tests for multiple senders / parallel senders as such.
So updated the set of receiver tests to exclude subscription related ones.

Reply:
There are other test cases that fall under the same category of pure client side receiver behavior and so we can keep either of the queue or subscription, but both are not needed.
- `onError handler is called for user error`
- `Second receive operation should fail if the first streaming receiver is not stopped`

Reply:
Agree, updated

Reply:
The stop() related case seems similar to error on close() cases (in that they rely on underlying AMQP implementation to handle the stop/close as expected)